app/models/payment_provider_config.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Die Mitte Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+# == Schema Information
+#
+# Table name: payment_provider_configs
+#
+#  id                        :integer          not null, primary key
+#  payment_provider          :string(255)
+#  invoice_config_id         :integer
+#  status                    :integer          default(0)
+#  partner_identifier        :string(255)
+#  user_identifier           :string(255)
+#  encrypted_password        :string(255)
+#  encrypted_keys            :text(16777215)
+#  synced_at                 :datetime
+#  created_at                :datetime
+#  updated_at                :datetime
+#
+
+class PaymentProviderConfig < ActiveRecord::Base
+  include Encryptable
+
+  enum status: [:draft, :pending, :registered]
```

Comment:
Im Ticket war `:new` und nicht `:draft` geplant. Geht allerdings nicht, da das versucht die `PaymentProviderConfig.new` Methode zu überschreiben...
Bessere Vorschläge als Synonym sind herzlich willkommen 🙃

Reply:
LOL. In dem Fall ist `:draft` auch okay. Es sind eben angelegte Verbindungen, die noch nicht aktiviert wurden.